project_event/models/project_task.py

```diff
@@ -189,12 +194,36 @@ def get_equipment_ids_inside(self):
         room_id = self.env['resource.calendar.room'].browse(self.room_id).id
         return room_id.instruments_ids.ids
 
+    @api.multi
+    def cancel_resources_reservation(self):
+        reservation_event = self.env['calendar.event'].browse(
```

Comment:
@eilst Please format the code for better readability e.g
 reservation_event = self.env['calendar.event'].\
browse(
e.g  def create(self, vals): in project_task model